### PR TITLE
Readme edited

### PR DESCRIPTION
**Pull request by sabrinatkk**
- Format and outline updated
- content edited
- spelling and grammar checked

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
 
 **Scope**
 
-At this stage, `Colourblind8` will focus on developing settings for the three most prevalent colourblind perspectives - protanopia, deuteranopes and tritanopes (see Appendix for more information on the types of colourblindness). 
+At this stage, `Colourblind8` will focus on developing settings for the three most prevalent colourblind perspectives - protanopia, deuteranopia and tritanopia (see [Appendix](https://github.com/UBC-MDS/Colourblind8#Appendix) for more information on the types of colourblindness).
 
 
-### Function Discription
+### Function Description
```
Comment (hfboyce) [inline README.md]: LOL OMG I SPELLED DESCRIPTION WRONG

File: README.md [hunk 2]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
+
+
+In this package, three functions are included to make `matplotlib`plots more interpretable for people with protanopia, deuteranopia and tritanopia. Within each function, users can select one of our customized colour palettes via the `palette` argument according to their preference: `deutera` for deuteranopia,`prota` for protanopia ,`trita`for tritanopia. The following are our main functions:
 
 - `plot_lines()`:
 
-  - This function implements a method that makes line graphs accesible to people with one of these three colourblind variants; deuteranopia, protanopia or tritanopia. It modifies the colour of each line, the title, transparency of the objects, legend and axes size as well as the face colour.
+  - This function implements a method that enhances line graphs through modifying the colour of lines, the face colour of the chart, the transparency of objects, and the font size for title, legend and axes.
```
Comment (hfboyce) [inline README.md]: This function implements a method that creates line graphs by initializing the colour of lines, the face colour of the chart, the transparency of objects, and the font size for title, legend and axes.

File: README.md [hunk 3]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
 
    **inputs:** x variable,  y variable (a list of lists), transparency, labels of each y variable, palette colour, title, x labels, y labels and legend title.    
-   **output:** a matplotlib object  
+   **output:** a matplotlib line graph
```
Comment (hfboyce) [inline README.md]: a matplotlib line graph object
(this need to be clear that they can add further amendments if they want)

File: README.md [hunk 4]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
 
    **inputs:** x variable,  y variable (a list of lists), transparency, labels of each y variable, palette colour, title, x labels, y labels and legend title.    
-   **output:** a matplotlib object  
+   **output:** a matplotlib line graph  
 
 
 - `plot_scatter()`:
 
-  - This function implements a method that makes scatterplots accesible to people with one of these three colourblind variants; deuteranopia, protanopia or tritanopia. It modifies the colour of each line, the title, legend and axes size, transparancy of the objects, as well as the face colour.
+  - This function implements a method that enhances scatterplots through modifying the colour of points, the face colour of the chart, the transparency of objects, and the font size for title, legend and axes.
```
Comment (hfboyce) [inline README.md]: This function implements a method that creates scatterplots by initializing the colour of lines, the face colour of the chart, the transparency of objects, and the font size for title, legend and axes.

File: README.md [hunk 5]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
 
    **inputs:** x variable,  y variable (a list of lists), transparency, labels of each y variable, palette colour, title, x labels, y labels and legend title.    
-   **output:** a matplotlib object.  
+   **output:** a matplotlib scatterplot.
```
Comment (hfboyce) [inline README.md]: a matplotlib scatterplot object

File: README.md [hunk 6]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
+
 
+- `plot_histogram()`:
 
-.- `plot_histogram()`:
+   - This functions implements a method that enhances histograms through modifying the colour of histogram layers, the face colour of the chart, the transparency of objects, and the font size for title, legend and axes.
 
-   - This functions implements a method that makes histograms accesible to people with one of these three colourblind variants; deuteranopia, protanopia or tritanopia. It modifies the colour of each line, the title, legend and axes size, transparancy of the objects number of bins,  as well as the face colour.
```
Comment (hfboyce) [inline README.md]: This function implement a method that creates histograms by initializing the colour of histogram layers, the face colour of the chart, the transparency of objects, and the font size for title, legend and axes

File: README.md [hunk 7]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
+### Usage
 
-To use our package it is necessary for all functions to import our package as well as matplotlib in the following manner. 
-```
-import matplotlib.pyplot as plt
-from colourblind8.colourblind8 import Colourblind8
-```
+Please note, for a more clear execution of our package, Varada Kolhatkar (UBC MDS instructor) suggested an additional document (jupyter notebook) to demonstrate how to use our package with a test dataset, along with our tests being passed. She found this a more beneficial method of visually comparing the output with the test cases.
+
+The demonstration can be found in the `Example` folder of this repository named [sample_usage.ipynb](https://github.com/UBC-MDS/Colourblind8/blob/master/example/Sample.ipynb).
```
Comment (hfboyce) [inline README.md]: `example`
 lowercase e

File: README.md [hunk 8]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
 
-The test dataset used for this example is the following
+1. Since `Colourblind8` is built on `matplotlib`, it is necessary to import `matplotlib`, as well as `Colourblind8`. Don't forget to initialize `Colourblind8` before using.
 
 ```
-x=[1,2,3,4]
-y_list=[[1,2,3,4],[3,5,7,9],[4,7,5,9],[1,4,6,8],[4,6,7,9],[1,3,4,8],[0,4,7,2],[-1,4,7,-7],[9,7,5,3]]
+import matplotlib.pyplot as plt
+from colourblind8.colourblind8 import Colourblind8
+cb = Colourblind8() #initialize
```
Comment (hfboyce) [inline README.md]: DELETE THIS LINE!! 

They can call it whatever they want.

File: README.md [hunk 9]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
 ```
+2. Select one of the available Colourblind8 graphs and one of our customized colour palettes for data visualization. We picked `plot_lines()` and `palette = "trita"` as an example:
 
-**Input**
+`cb.plot_lines(self, x, y, alpha = 1.0, labels = None, palette = "trita", title = None, x_lab = None, y_lab = None, legend_title = None)`
 
-For the *deuteranopia* friendly colour palette, it is necessary to use the following commands for a line graph:
+Three graph choices are available: `plot_line()`,`plot_scatter()`,`plot_histogram()`
```
Comment (hfboyce) [inline README.md]: Delete this line

Comment (hfboyce) [reply README.md]: This isnt useful here

File: README.md [hunk 10]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
+y_list= []
+for i in range(9):
+    y = np.random.rand(N)
+    y_list.append(y)
 ```
-
-**Input**
-
-For the *deuteranopia* friendly colour palette, it is necessary to use the following commands for a scatterplot:
+code:
```
Comment (hfboyce) [inline README.md]: dont need to write this .

File: README.md [hunk 11]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
+###### Line Graph's input
+data:
 ```
+x=[1,2,3,4]
 
-cb = Colourblind8()
-cb.plot_line(x=x, y=y_list,alpha =1.0, labels =['a','b','c','d','e','f','g','h','i'], palette = 'prota', title = "Example Line", x_lab = "X" , y_lab = "Y" , legend_title = "Labels" )
+y_list=[[1,2,3,4],[3,5,7,9],[4,7,5,9],[1,4,6,8],[4,6,7,9],[1,3,4,8],[0,4,7,2],[-1,4,7,-7],[9,7,5,3]]
 ```
+code:
```
Comment (hfboyce) [inline README.md]: dont need to write this

File: README.md [hunk 12]
```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
...
 y = np.random.rand(N)
 z = np.random.rand(N)
 list_y = [x,y,z]
-
 ```
-
-**Input**
-
-For the *deuteranopia* friendly colour palette, it is necessary to use the following commands for a scatterplot:
+code:
```
Comment (hfboyce) [inline README.md]: delete

**Comment by sabrinatkk** [comment]
> This isnt going to format properly i dont think....

I checked the format in Markdown Revew in Atom, it worked. But @hfboyce you can check it again

**Comment by sabrinatkk** [comment]
> This isnt going to format properly i dont think....

I checked the changes in Atom markdown review, it worked.

File: README.md (hunk 3 above)
Comment (sabrinatkk) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (sabrinatkk) [reply README.md]: change made to new Readme.

File: README.md (hunk 5 above)
Comment (sabrinatkk) [reply README.md]: change made to new Readme.

File: README.md (hunk 6 above)
Comment (sabrinatkk) [reply README.md]: change made to new Readme.

File: README.md (hunk 7 above)
Comment (sabrinatkk) [reply README.md]: change made to new Readme.

File: README.md (hunk 8 above)
Comment (sabrinatkk) [reply README.md]: change made to new Readme.

File: README.md (hunk 9 above)
Comment (sabrinatkk) [reply README.md]: change made to new Readme.

File: README.md (hunk 11 above)
Comment (sabrinatkk) [reply README.md]: change made to new Readme.

File: README.md (hunk 10 above)
Comment (sabrinatkk) [reply README.md]: change made to new Readme.

File: README.md (hunk 8 above)
Comment (hfboyce) [reply README.md]: cb = Colourblind8() #initialize just this one

**Pull request merged**